layers/odbc/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG PHP_VERSION
+FROM bref/build-php-$PHP_VERSION AS ext
+
+ENV LD_LIBRARY_PATH=/usr/lib:/usr/lib64:$LD_LIBRARY_PATH
+
+# Install packages
+RUN yum install -y php-odbc
```

Comment:
Aren’t you missing a pecl install?

Reply:
i can't install odbc via pecl, because this ended with error

Reply:
![image](https://user-images.githubusercontent.com/12143866/72607728-78b42f80-3921-11ea-9987-e87a13047fcb.png)